R/deploymentTarget.R

```diff
@@ -72,6 +70,29 @@ deploymentTarget <- function(recordPath = ".",
   }
 }
 
+deploymentTargetForApp <- function(appId,
+                                   appTitle = NULL,
+                                   account = NULL,
+                                   server = NULL) {
+  accountDetails <- accountInfo(account, server)
+  client <- clientForAccount(accountDetails)
```

Comment:
aside: `clientForAccount` might be the right place for the URL validation pre-flight we chatted about, even though we might construct multiple client objects during the deploy workflow.